backend/models/projectInfo.test.js

```diff
@@ -0,0 +1,90 @@
+const mongoose = require('mongoose');
+const { Location } = require('./dictionaries/location.model');
+const { ProjectInfo } = require('./projectInfo.model');
+const { setupDB } = require("../setup-test");
+
+setupDB("projectInfo-model");
+
+describe("ProjectInfo Model saves the correct values", () => {
+  test("Save a model instance and then read from the db", async (done) => {
+    
+    const testData = {
+      locations: ["location1","location2"]
+    };
+ 
+    await Location.create(testData);
+    const test = await Location.find({});
+
+    const submittedData = {
+      name: "projectTest",
+      description: "An instance of a Project Info model",
+      projectStatus: "Active", // Active, Completed, or Paused
+      locationZone: "location1", 
+      createdDate: 1594023390039, // date/time project was created
+      completedDate: 1594023390039, // only if Status = Completed, date/time completed
+      urls: {repository: "https://github.com/hackforla/VRMS"}, // link to main repo
+      slackUrl: "hackforla.slack.com", // link to Slack channel
+    };
+
+    await ProjectInfo.create(submittedData);
+    const savedDataArray = await ProjectInfo.find();
+    const savedData = savedDataArray[0];
+    expect(savedData.name).toBe(submittedData.name);
+    expect(savedData.urls.repository).toBe(submittedData.urls.repository);
+
+
+    done();
+  });
+});
+
+describe('CREATE/READ', () => {
+  test('Create ProjectInfo with Mongoose model', async (done) => {
+    // const submittedData = {
```

Comment:
Commenting out tests is not a fix. Tests failed because of some code issues not because they are bad.

Reply:
@alex-anakin I actually did not comment that out. Those were originally commented out in https://github.com/hackforla/VRMS/pull/493/files

Reply:
I have uncommented the pertinent section as requested